functions/src/submitApplication/models.js

```diff
@@ -3,12 +3,14 @@ const cleaner = require('deep-cleaner');
 exports.Hacker = function Hacker(data) {
   const hacker = {
     hacker_quick_info: {
+      id: data.id,
       firstName: data.firstName,
       lastName: data.lastName,
       email: data.email,
       timestamp: data.timestamp,
     },
     hacker_main_info: {
+      id: data.id,
```

Comment:
Can we also add `hacker_quick_info` attributes to `hacker_main_info`? and maybe rename `hacker_main_info` to `hacker_full_info`?